npm/private/test/yaml_tests.bzl

```diff
@@ -126,6 +126,81 @@ def _parse_basic_test_impl(ctx):
 }
 """))
 
+    # Literal multiline strings (|), strip (-) and keep (+)
+    asserts.equals(env, {"foo":"bar\n" }, parse("""\
+foo: |
+    bar
```

Comment:
would a trailing spaces after `bar` be kept?

Reply:
Yes, they should be. I'll add a test case for that.

Reply:
Added some trailing spaces to the third test case after `baz`. You can't see those spaces in the github diff, but you can see it in the expected value.

Reply:
🌮